templates/taxi/driver_list.html

```diff
@@ -7,7 +8,10 @@ <h1>
         Create
       </a>
     </h1>
-
+    <form method="get" action="" class="form-inline">
+    {{ search_form|crispy }}
+    <input type="submit" class="btn btn-secondary" value="😈">
```

Comment:
The submit button value '😈' might not be appropriate for a professional application. Consider using a more descriptive label like 'Search'.